internal/impl/io/input_websocket.go

```diff
@@ -43,6 +43,9 @@ func websocketInputSpec() *service.ConfigSpec {
 			service.NewStringField("open_message").
 				Description("An optional message to send to the server upon connection.").
 				Advanced().Optional(),
+			service.NewStringListField("open_messages").
+				Description("An optional messages to send to the server upon connection.").
```

Comment:
Better to say that "if `open_message` is set at the same time, `open_messages` will be sent after the `open_message` is sent.".

Reply:
you are right, it is not inclusive now, user might add both